examples/nodejs-alpha/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "dependencies": {
+    "@google-cloud/agones-sdk": "file:../../sdks/nodejs"
```

Comment:
This is because the npm module is not published yet and should be fine for local testing